third_party/move/move-binary-format/src/errors.rs

```diff
@@ -330,10 +330,19 @@ impl PartialVMError {
     }
 
     pub fn new(major_status: StatusCode) -> Self {
+        debug_assert!(major_status != StatusCode::EXECUTED);
+        let message = if major_status == StatusCode::UNKNOWN_INVARIANT_VIOLATION_ERROR {
+            Some(format!(
+                "UNKNOWN_INVARIANT_VIOLATION_ERROR at {:?}",
+                std::backtrace::Backtrace::force_capture()
```

Comment:
Let's probably limit the number of frames we record using https://docs.rs/backtrace/latest/backtrace/

Reply:
Done